Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizer.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Demo class to show different ways of using the boundary API.
+    /// </summary>
+    public class BoundaryVisualizer : MonoBehaviour
```

Comment:
wouldn't we want something like this in the SDK?

Reply:
not in this form... but we should consider one that shows the playarea quad (inscribed rect)

Reply:
renaming this script and class to be clear it is a demo